lib/optimist.rb

```diff
@@ -84,7 +84,11 @@ def self.registry_getopttype(type)
   ##  ignore options that it does not recognize.
   attr_accessor :ignore_invalid_options
 
-  DEFAULT_SETTINGS = { suggestions: true, exact_match: true }
+  DEFAULT_SETTINGS = {
+    exact_match: true,
+    explicit_short_opts: false,
```

Comment:
I'm finding the "negative" option text, a default of false, and then using `unless` in the code with it very confusing (like a triple negative in my mind).  Perhaps it's better to flip the logic to the positive with:

```suggestion
    implicit_short_opts: true,
```

Then later, this reads better to me:

```
-     resolve_default_short_options! unless @settings[:explicit_short_opts]
+     resolve_default_short_options! if @settings[:implicit_short_opts]

Reply:
This would also set all the default settings to "true" and making them all "opt-out", so it's consistent.

Reply:
I'll switch the polarity from `explicit_short_opts: false` to `implicit_short_opts: true` and push it up